test/test_pebble.py

```diff
@@ -283,8 +286,8 @@ def test_task_init(self):
         self.assertEqual(task.data, {})
 
     def test_task_from_dict(self):
-        d = {
-            "id": "78",
+        d: pebble._TaskDict = {
+            "id": typing.cast(pebble.TaskID, "78"),
```

Comment:
I believe this field should be a regular string, as it's in the raw argument to `from_dict`.

Reply:
It has to be a `TaskID`, because we're passing in a `_TaskDict` and the `id` in `_TaskDict` is a `TaskID`. Type checkers complain if it's not a `TaskID`.

Fwiw, it seems strange to me that type checkers aren't ok with a string literal being used where a string type alias is expected. But that's the current state.

Reply:
Oops, yeah, that `id` field should not be a `TaskID`, just a regular `str`, as that type is a plain dict that comes from the wire from the Pebble API.